src/zenml/zen_stores/schemas/pipeline_schemas.py

```diff
@@ -79,13 +78,15 @@ class PipelineSchema(NamedSchema, table=True):
         nullable=True,
     )
 
+    # Relationships
     user: Optional["UserSchema"] = Relationship(back_populates="pipelines")
+    workspace: "WorkspaceSchema" = Relationship(back_populates="pipelines")
 
     schedules: List["ScheduleSchema"] = Relationship(
         back_populates="pipeline",
     )
     runs: List["PipelineRunSchema"] = Relationship(
-        back_populates="pipeline", sa_relationship_kwargs={"cascade": "delete"}
+        sa_relationship_kwargs={"cascade": "delete"}
     )
```

Comment:
Shouldn't this have a relationship to deployments now instead and cascade-delete all pipeline deployments which in turn cascade-delete all of their runs?

Reply:
You are right, fixed the relationship.